oldlog.go

```diff
@@ -121,7 +121,7 @@ func GetSubsystems() []string {
 
 func getLogger(name string) *logging.Logger {
 	log := logging.MustGetLogger(name)
-	log.ExtraCalldepth = 1
+	log.ExtraCalldepth = 0 
```

Comment:
Did you run gofmt?  There is an extra space after the 0.

Reply:
I didn't run gofmt. I just found this problem in my machine and ipfs.exe version 0.4.15.
If this problem exist on the other platform. You can fix it.